helix-term/src/commands/typed.rs

```diff
@@ -3008,6 +3105,13 @@ pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
         fun: redraw,
         signature: CommandSignature::none(),
     },
+    TypableCommand {
+        name: "trim",
+        aliases: &[],
+        doc: "Trim whitespace",
```

Comment:
The description of the command could be more specific(longer), until you try to use the command it's not clear whether it trims the buffers' content, or just the selection.

Also, the word "trim" is used with another meaning than in `trim_selection`, how could we differentiate these two?
* trim: to move the selection anchors so that the selection no longer includes undesirable characters, but these characters still exist.
* trim: to delete characters from the buffer, if they are undesirable.